src/styles/blocks/reviews.css

```diff
@@ -0,0 +1,36 @@
+.card-reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card-reviews__stars {
+  display: flex;
+  align-items: center;
+}
+
+.card-reviews__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 16px;
+
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card-reviews__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.card-reviews__stars .card-reviews__star:nth-child(-n+4) {
```

Comment:
See checklist (item 3) - add other selectors as in homework with Stars. Because if i want 3 stars, for example, i can add modifier to block with stars and count of active stars should change.